JASP-Engine/JASP/R/auditClassicalBenfordsLaw.R

```diff
@@ -591,9 +591,9 @@ auditClassicalBenfordsLaw <- function(jaspResults, dataset, options, ...){
   distribution <- base::switch(options[["distribution"]], "benford" = "Benford's law", "uniform" = "the uniform distribution")
 
   conclusionText <- base::switch(options[["digits"]],
-                                  "first" = gettextf("The <i>p</i> value is determined to be %1$s. Therefore, the null hypothesis that the distribution of first numbers in the population conforms to %2$s <b>%3$s</b> with <b>%4$s</b> confidence.", pvalue, distribution, conclusion, confidenceLabel),
-                                  "firstSecond" = gettextf("The <i>p</i> value is determined to be %1$s. Therefore, the null hypothesis that the distribution of the two first numbers in the population conforms to %2$s <b>%3$s</b> with <b>%4$s</b> confidence.", pvalue, distribution, conclusion, confidenceLabel),
-                                  "last" = gettextf("The <i>p</i> value is determined to be %1$s. Therefore, the null hypothesis that the distribution of last numbers in the population conforms to %2$s <b>%3$s</b> with <b>%4$s</b> confidence.", pvalue, distribution, conclusion, confidenceLabel))
+                                  "first" = gettextf("The <i>p</i> value is determined to be %1$s. Therefore, the null hypothesis that the distribution of first digits in the data set conforms to %2$s <b>%3$s</b> with <b>%4$s</b> confidence.", pvalue, distribution, conclusion, confidenceLabel),
+                                  "firstSecond" = gettextf("The <i>p</i> value is determined to be %1$s. Therefore, the null hypothesis that the distribution of the two first digits in the data set conforms to %2$s <b>%3$s</b> with <b>%4$s</b> confidence.", pvalue, distribution, conclusion, confidenceLabel),
```

Comment:
Determined in this context is sounds a bit too deterministic. 

"The <i>p</i> value is **observed** to be %1$s. "

Reply:
Will do